cl/opinion_page/forms.py

```diff
@@ -33,13 +33,15 @@ class CitationRedirectorForm(forms.Form):
         widget=forms.TextInput(
             attrs={"class": "form-control input-lg", "placeholder": "Volume"}
         ),
-        required=True,
+        required=False,
     )
+    # We change the place holder to allow people to continue to use the rest
+    # of the redirect API with no modifications.
     reporter = forms.CharField(
         widget=forms.TextInput(
             attrs={
                 "class": "form-control input-lg",
-                "placeholder": "Reporter",
+                "placeholder": "Full Citation",
```

Comment:
I think we can do better here. How about:

```suggestion
                "placeholder": "Paste any text containing a citation…",
```